tensorflow_datasets/structured/letter_recognition_test.py

```diff
@@ -0,0 +1,23 @@
+"""Database of character image features; try to identify the letter."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.structured import letter_recognition
+
+class LetterRecognitionTest(testing.DatasetBuilderTestCase):
+
+  DATASET_CLASS = letter_recognition.LetterRecognition
+  SPLITS = {
+
+      "train": 1,
+
+  }
+
+  DL_EXTRACT_RESULT = {'letter-recognition.data'}
```

Comment:
Please test your code! You are passing in a type `set` (the dictionary) as the `DL_EXTRACT_RESULT` variable.